src/components/WordGroupSelector/squiggly.svg

```diff
@@ -0,0 +1,16 @@
+<svg width="1500" height="1500" viewBox="0 0 1319 1062" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
if this is the squiggly for the background maybe name it wordgroupbg or something instead of squiggly LOL